src/components/Fields/TextArea.test.js

```diff
@@ -304,4 +304,46 @@ describe('TextArea', () => {
         const textField = shallow(<TextArea />);
         expect(textField.hasClass('uir-text-area--has-value')).to.not.equal(true);
     });
+
+    it('fixes input focus when it\'s wrapped by tooltip', () => {
+        const textField = mount(<TextArea isValid tooltipError="Error" />);
+        textField.setState({ hasFocus: true });
+        sinon.spy(textField.instance(), 'fixInputFocus');
+
+        textField.setProps({
+            isValid: false,
+        });
+        expect(textField.instance().fixInputFocus).to.be.called();
+    });
+
+    it('fixes input focus when it ceases to be wrapped by a tooltip', () => {
+        const textField = mount(<TextArea isValid={false} tooltipError="Error" />);
+        textField.setState({ hasFocus: true });
+        sinon.spy(textField.instance(), 'fixInputFocus');
+
+        textField.setProps({
+            isValid: true,
+        });
+        expect(textField.instance().fixInputFocus).to.be.called();
+    });
+
+    it('fixInputFocs calls input ref focus', () => {
```

Comment:
Typo:
```suggestion
    it('fixInputFocus calls input ref focus', () => {
```

Could you also fix this in the `TextField` tests?

Reply:
Changed